Paws4Me/Paws4Me/Show and Validate Login/View Controllers/SignInViewController.swift

```diff
@@ -34,28 +31,21 @@ class SignInViewController: UIViewController, Validation {
 
     // MARK: - IBActions
     @IBAction private func signInButtonTapped (_ sender: UIButton!) {
-        guard let username = usernameTextField.text, let password = passwordTextField.text else { return }
-        isLoggedIn = isValidCredentials(username: username, password: password)
         var errorMessage = ""
+        guard let username = usernameTextField.text, let password = passwordTextField.text else { return }
+        isLoggedIn = signInViewModel.isValidCredentials(username: username, password: password)
+
         if isLoggedIn {
```

Comment:
Why not have a delegate protocol and let your `ViewModel` decide what should happen next 😄

Reply:
Will do so 👍🏻